presto-docs/src/main/sphinx/optimizer/push-down.rst

```diff
@@ -0,0 +1,140 @@
+=========
+Push down
+=========
+
+Presto can push down processing queries, or segments of queries, down to the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database for processing.
+
+The results of this push down can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the database
+* reduced load on the database
+
+Support for push down is specific to each connector and the relevant underlying
+database or storage system.
+
+Every Presto aggregate function is eligible for push down, as long as it
+operates on types supported by the given connector.
+
+Analysis and Confirmation
+-------------------------
+
+Push down depends on a number of factors:
+
+* generic support for push down in the connector
+* function or operation specific support for push down in the connector
+* query that allows the detection of the function to push down
+
+The best way to analyze if push down for a specific query is performed, is to
+take a closer look at the :doc:`EXPLAIN plan </sql/explain>` of the query. If an
+operations such as an aggregate function successfully pushed down to the
+connector, the explain plan does **not** show that operator. The explain plan
+only shows the operations that are performed by Presto.
+
+Here is an example query using the TPCH data, which was pushed into a PostgreSQL
+database and it now queried with the PostgreSQL connector::
+
+    USE postgresql.tpch.tiny;
```

Comment:
Let's remove this `USE` statement, since it's explained in the above that this is querying the PostgreSQL connector. It's not something a user could use as-is without full steps to load into exactly this schema and table.

Reply:
Sure .. having it in the example just makes it more explicit ... personally I would leave it but I am removing